modules/ocf_desktop/files/xsession/notify

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import datetime
+import sched
+
+from gi.repository import Notify
+from ocflib.lab.stats import staff_in_lab
+from ocflib.misc.whoami import current_user
+from ocflib.printing.quota import get_connection
+from ocflib.printing.quota import get_quota
+
+
+NOTIFICATION_ICON = '/opt/share/xsession/images/ocf-color.png'
+
+# Format taken from paper command
+QUOTA_FORMAT = """\
+    → {semesterly} remaining this semester
+    → {daily} remaining today"""
```

Comment:
Nah, the color formatting used by paper only works in a shell, plus we already get the username and quota so we can tell if it changed. The only code copied is the use of the "→" character.